src/firebase/firestore/transaction/index.js

```diff
@@ -0,0 +1,21 @@
+export default class TransactionWriteBatch {
+  delete(ref) {
+    return ref.delete();
+  }
+
+  set(ref, data, option = {}) {
+    return ref.set(data, option);
+  }
+
+  update(ref, data) {
+    return ref.update(data);
+  }
+
+  get(ref) {
+    return ref.get();
+  }
+
+  getAll(...refs) {
```

Comment:
I don't see this as a public API [here](https://firebase.google.com/docs/reference/js/firebase.firestore.Transaction). Should we remove this?

Reply:
I actually checked [firestore client](https://github.com/googleapis/nodejs-firestore/blob/master/dev/src/transaction.ts#L188-L207) code to copy the API, now I saw that it is private. Will remove.